docs/persistency.md

```diff
@@ -160,3 +160,117 @@ This is equivalent to the following command executed inside the container:
 ```shell
 docker exec -it yaptide_flask ./yaptide/admin/db_manage.py list-users
 ```
+
+## Developing model
+
+In Yaptide flask-migrate is responsible for modyfing database after each change to `models.py` and keeping track of versions of database (new version comes after each modification of models.py).
+
+### Development steps
+For development - running yaptide_postgres in docker is required. It's recommended to do development on local machine.
```

Comment:
in principle flask migrate can be handled as well to make changes in any DB, like sqlite, but it doesn't make much sense, as we use sqlite only for local dev